Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM maven:latest
+
+RUN useradd -m -s /bin/bash amnotbot
+RUN mkdir -p /home/amnotbot/.amnotbot
+
+COPY src/main/resources/*.config /home/amnotbot/.amnotbot/
+COPY src/main/resources/ignore.words /home/amnotbot/.amnotbot/
+COPY src/main/resources/log4j.properties /home/amnotbot/.amnotbot/
+ADD . /home/amnotbot/app
+
+RUN chown -R amnotbot:amnotbot /home/amnotbot
+
+USER amnotbot
+ENV HOME /home/amnotbot
+
+WORKDIR /home/amnotbot/app
+
+RUN mvn -e install -DskipTests
+
+CMD ["java","-jar","target/amnotbot-core-0.0.1-SNAPSHOT.jar"]
```

Comment:
I tried `ENV` and `ARG` but could not make it work. `ENV` is only visible inside the container, I understand and not during the building process. And `ARG` you need to pass the value when building the image.

Tried something like:

`ENV AMNOTBOT_VERSION 0.0.1-SNAPSHOT`
`CMD ["java","-jar","target/amnotbot-core-$AMNOTBOT_VERSION"]`

But without success (in docker for openSUSE 42.2).